src/applications/vaos/services/location/transformers.js

```diff
@@ -159,6 +159,10 @@ export function transformFacility(facility) {
           facility.uniqueId
         }`,
       },
+      {
```

Comment:
Not sure if this will break things

Reply:
It did, but now it doesn't because I'm using the test facility id. I think we need to clean up some of this test facility id conversion